jdaviz/tests/test_utils.py

```diff
@@ -1,20 +1,80 @@
 import os
+import pytest

```

Comment:
As per `isort`, `pytest` belongs in "third-party" section, not "stdlib".
```suggestion
```

Reply:
I think this happened from a rebase but I am also not familiar with the order of imports. Now looking at several files is the standard practice here python distro, third party, and then STScI? 

I will make the changes to each of the import comments!